src/uploadx/lib/store.ts

```diff
@@ -1,16 +1,24 @@
 class Store {
   prefix = 'UPLOADX-V3.0-';
 
-  set(key: string, value: string): void {
+  set(key: string, value: string): this {
     localStorage.setItem(this.prefix + key, value);
+    return this;
   }
 
   get(key: string): string | null {
     return localStorage.getItem(this.prefix + key);
   }
 
-  delete(key: string): void {
+  delete(key: string): boolean {
```

Comment:
Why make `set` chainable and not `delete`? Returning always `true` is just as good as returning undefined, no?